.teamcity/settings.kts

```diff
@@ -0,0 +1,76 @@
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.PullRequests
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.pullRequests
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.powerShell
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.vcs
+
+project {
+    buildType(ChocolateyAU)
+}
+
+object ChocolateyAU : BuildType({
+    id = AbsoluteId("ChocolateyAU")
+    name = "Build"
+
+    artifactRules = """
+        +:code_drop/Chocolatey-AU/**
+        +:code_drop/*.nupkg
+        +:code_drop/*.7z
+        +:code_drop/test.results.xml
+    """.trimIndent()
+
+    params {
+        param("env.vcsroot.branch", "%vcsroot.branch%")
+        param("env.Git_Branch", "%teamcity.build.vcs.branch.ChocolateyAu_HttpsGithubComCorbobChocolateyAuRefsHeads32buildPipeline%")

```

Comment:
```suggestion
        param("env.Git_Branch", "%teamcity.build.vcs.branch.ChocolateyAU_ChocolateyAUVcsRoot%")
```

Just making this match what the change on the Ops side of things dictates this will be.

Reply:
Yeah, The initial setting was what the dev Team City server decided to call it. I've redone the dev Team City configuration to match this so we can see the builds working there.